opencti-platform/opencti-graphql/config/default.json

```diff
@@ -221,7 +221,8 @@
     "max_tokens": 30000,
     "insights_refresh_timeout": 60,
     "ai_azure_instance": "",
-    "ai_azure_deployment": ""
+    "ai_azure_deployment": "",
+    "ai_timeout": 30000
```

Comment:
```suggestion
    "timeout": 60000
```

Reply:
I think timeout is enough, or en var it will be AI__TIMEOUT (or else with first proposal it would be AI__AI_TIMEOUT)